hw/ip/otbn/data/otbn.hjson

```diff
@@ -102,30 +102,79 @@
         }
       ]
     } // register : status
-    { name: "ERR_CODE",
-      desc: "Error Code",
+    { name: "ERR_BITS",
+      desc: '''
+        Error bitfield. Reads as non-zero if an error was seen during OTBN
+        operation
+      ''',
       swaccess: "ro",
       hwaccess: "hwo",
       fields: [
-        { bits: "31:0",
-          name: "err_code",
+        { bits: "0",
+          name: "bad_data_addr"
           desc: '''
-            The error cause if an error occurred.
-
-            This is set to zero (ErrCodeNoError) when OTBN successfully
-            completes an operation or a positive value when OTBN stops on an
-            error.
-
-            While OTBN is running (!!CMD.busy set), this has no meaning. When
-            OTBN stops this will be set to indicate the outcome. It is
-            guaranteed not to change until OTBN is run again (with !!CMD.start).
-
-            See <a href='#error-conditions'>Error Conditions</a> for
-            details on the possible values.
+            A DMEM read or write occurred with an out of bounds or unaligned
+            address.
           '''
         }
+        { bits: "1",
+          name: "bad_insn_addr"
```

Comment:
It seems to be common to use uppercase letters for those constants, even though not universal.

Reply:
I left it as lower case as the other fields in OTBN registers are also lower case. We could switch them all to upper but I think we're fine leaving them.